src/Symfony/Component/HttpKernel/Controller/ArgumentResolverInterface.php

```diff
@@ -24,7 +24,6 @@ interface ArgumentResolverInterface
     /**
      * Returns the arguments to pass to the controller.
      *
-     * @param Request  $request
      * @param callable $controller
```

Comment:
Couldn't move `callable` doc type to type hint, I got type errors as it's considered an array here:

https://github.com/symfony/symfony/blob/677ed339f4eccd10b94d4bc4cbf48bbfbfa5d81b/src/Symfony/Component/HttpKernel/Tests/Controller/ArgumentResolverTest.php#L181.L195

Reply:
Arrays can fullfil the callable type declaration. To me it looks like the test is broken: The `controller*` methods need to be public.

Reply:
That's why indeed, good catch